compiler/onnx-tools/onnx-dump.py

```diff
@@ -0,0 +1,154 @@
+        # TODO revise when graph node is available
+        return "<graph>"
+    if attr.type == AttributeProto.TENSORS:
+        # TODO revise to see contents
+        return "<tensors>..."
+    if attr.type == AttributeProto.GRAPHS:
+        # TODO revise when graph node is available
+        return "<graphs>..."
+    return helper.get_attribute_value(attr)
+
+
+def _dump_header(onnx_model):
+    print("[General] -----------------------------")
+    print("IR version =", onnx_model.ir_version)
+    print("Producer   =", onnx_model.producer_name, onnx_model.producer_version)
+    print("")
+
+
+def _dump_operators(onnx_model):
+    opcodes_dict = dict()
+    for node in onnx_model.graph.node:
+        if node.op_type in opcodes_dict:
+            opcodes_dict[node.op_type] = opcodes_dict[node.op_type] + 1
+        else:
+            opcodes_dict[node.op_type] = 1
+
+    print("[Operators] ---------------------------")
+    for opcode_key in opcodes_dict:
+        print("{:>5} {}".format(opcodes_dict[opcode_key], opcode_key))
+
+    print("")
+
+
+def _dump_initializers(onnx_model):
+    print("[Initializers] ------------------------")
+    for initializer in onnx_model.graph.initializer:
+        dims = []
+        idx = 0
+        for d in initializer.dims:
+            dims.append(d)
+        init_name = '"{}"'.format(initializer.name)
+        dtstr = _data_type_str(initializer.data_type)
+        print('{:<15} {} {}'.format(init_name, dtstr, dims))
+
+    print("")
+
+
+def _dump_nodes(onnx_model):
+    print("[Nodes] -------------------------------")
+
+    for node in onnx_model.graph.node:
+        print('{0}("{1}")'.format(node.op_type, node.name))
+
+        attribute = ''
+        for attr in node.attribute:
+            if attribute != '':
+                attribute += ', '
+            attribute += "{}: {}".format(attr.name, _get_attribute_value(attr))
+
+        if attribute != '':
+            print('    A {0}'.format(attribute))
+
+        for inp in node.input:
+            print('    I "{0}"'.format(inp))
+        for out in node.output:
+            print('    O "{0}"'.format(out))
+
+    print("")
+
+
+def _dump_inputoutputs(onnx_model):
+    print("[Graph Input/Output]-------------------")
+    for mod_input in onnx_model.graph.input:
+        io_name = '"{}"'.format(mod_input.name)
+        dtstr = _data_type_str(mod_input.type.tensor_type.elem_type)
+        shape = mod_input.type.tensor_type.shape
+        input_shape = []
+        for dim in shape.dim:
+            input_shape.append(dim.dim_value)
+        print('    I: {:<15} {} {}'.format(io_name, dtstr, input_shape))
+
+    for mod_output in onnx_model.graph.output:
+        io_name = '"{}"'.format(mod_output.name)
+        dtstr = _data_type_str(mod_output.type.tensor_type.elem_type)
+        shape = mod_output.type.tensor_type.shape
+        output_shape = []
+        for dim in shape.dim:
+            output_shape.append(dim.dim_value)
```

Comment:
(optional) Pythonic style :)
```suggestion
        output_shape = [dim.dim_value for dim in shape.dim]
```